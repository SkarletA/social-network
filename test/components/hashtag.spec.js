import loadHashtag from '../../src/components/hashtag.js';

jest.mock('../../src/firebase.js');

describe('loadHashtag', () => {
  document.body.innerHTML = '<div id="root"></div>';
  test('render del componente de hashtags', () => {
    const root = document.getElementById('root');
    root.appendChild(loadHashtag());
    expect(root.innerHTML).toMatchSnapshot();
  });
});
