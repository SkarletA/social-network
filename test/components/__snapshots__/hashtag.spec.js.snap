// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loadHashtag render del componente de hashtags 1`] = `"<div class=\\"hashContainer\\"></div>"`;
